ocw/lib/EC2.py

```diff
@@ -13,6 +13,12 @@
 
 class EC2(Provider):
     __instances = dict()
+    default_region = 'eu-central-1'
+
+    def __init__(self, namespace: str):
```

Comment:
Hm I'm not sure anymore why I didn't used a `__init__()` function here. But I gues it has something to do with that singleton approach.
Why you don't put this also into the `__new__`?